config.json

```diff
@@ -882,9 +882,11 @@
       "slug": "variable-length-quantity",
       "core": false,
       "unlocked_by": null,
-      "difficulty": 1,
+      "difficulty": 6,
       "topics": [
-
+        "Integers",
+        "Bitwise operations",
+        "Transforming"
```

Comment:
I would add "Arrays".  The Go term is Slices, but we are picking from the [Topics](https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt).